### PR TITLE
Use Babel package doc

### PR DESCRIPTION
**Pull request by xtuc**
Refers to issue #528.

NPM package page shows the header but the website already have one. I had to write a custom plugin to drop these lines. It uses the `page.package` to determine its README location.

I assumed that the header (package title and description) takes three lines.

There is a git module to include babel package repository.

This PR shows only a demo. Many documentations from the babel repo and from the website are very different.

### CODE REVIEW

**Comment by hzoo** [comment]
Sounds like a good start! - will check it out later

File: _plugins/markdown.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+=begin
```
Comment (thejameskyle) [inline _plugins/markdown.rb]: GitHub Pages will not build with any plugins, so you don't even have the option to use it

File: _includes/package_readme.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{% capture readme %}
+    {% include babel/packages/{{ page.package }}/README.md %}
+{% endcapture %}
+
+{{ readme
+    | newline_to_br
+    | split: "<br />"
+    | slice: 4, 1000
```
Comment (xtuc) [inline _includes/package_readme.html]: I also can use `shift` there.

Like this : 
```
    | split: "<br />"
    | shift
    | shift
    | shift
    | shift
    | join: "<br />"
```

This way I don't need to limit the string to 1000 but I don't like it.
Do you mind @thejameskyle ?

File: .gitmodules [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[submodule "_includes/babel"]
```
Comment (hzoo) [inline .gitmodules]: We'll need to add some instructions to the readme or add that automatically in make serve/npm start?

File: _includes/package_readme.html (hunk 2 above)
Comment (hzoo) [inline _includes/package_readme.html]: This is to remove the first 4 header lines at the top of every readme https://raw.githubusercontent.com/babel/babel/master/packages/babel-plugin-syntax-async-generators/README.md - they all ahve it so it should be ok as a workaround (using shift).

**Comment by hzoo** [comment]
LGTM we can make an issue or just iterate on the whole remove first 4 lines thing

**Pull request merged**